src/inventory/inventory.resolver.ts

```diff
@@ -49,7 +53,7 @@ export class InventoryResolver {
     @Args() args: FindProductItemArgs,
     @Info() info,
   ): Promise<IPaginatedType<ProductItem>> {
-    console.log('Resolving productItems for ', args)
+    this.logger.debug(`Resolving productItems for ${JSON.stringify(args)}`)
```

Comment:
I am a bit weirded out that everything is just a debug log.
Does your framework already produce a `info` log for every request?
If not, please add it.
Every request that reaches the server needs at least one `info` log.